core/shared/src/main/scala/scalaz/zio/syntax/IOSyntax.scala

```diff
@@ -36,9 +36,15 @@ object IOSyntax {
 
   final class IOIterableSyntax[E, A](val ios: Iterable[IO[E, A]]) extends AnyVal {
     def mergeAll[B](zero: B, f: (B, A) => B): IO[E, B] = IO.mergeAll(ios)(zero, f)
-    def parAll: IO[E, List[A]]                         = IO.parAll(ios)
+    def collectPar: IO[E, List[A]]                     = IO.collectPar(ios)
     def forkAll: IO[Nothing, Fiber[E, List[A]]]        = IO.forkAll(ios)
-    def sequence: IO[E, List[A]]                       = IO.sequence(ios)
+    def collect: IO[E, List[A]]                        = IO.collect(ios)
+
+    /**
+     * Sequentially collect the results to a List.
+     * Named `collectIos` instead of just `collect` to avoid naming conflicts.
+     */
+    def collectSeq: IO[E, List[A]] = IO.collect(ios)
```

Comment:
This is the same as `collect` and can be deleted.

Reply:
`collectSeq` is required, because sometimes, for example in our very tests, `collect` is not available due to name clashes with other "extension methods"

Reply:
I see the collect above, but it's the same method. Why have two things that do the same thing? Better to delete the duplicate.

Reply:
sorry, perhaps it's difficult for me to explain... :D
there's a test for this `ios.collect-like-syntax` and it doesn't compile if it's like `ios.collect`, most likely because of naming conflicts
it compiles fine if it's `ios.collectSeq`

Reply:
I understand, but what I am saying is that there are two collect methods in the same scope, with the same name, and same types. So this alone will lead to clashes. I think you delete `collectSeq` it should work—or maybe you can show me the errors so I can figure out what's going on?

Reply:
if I keep the original test like this
```scala
  def t4 = {
    val ios = TestData.map(IO.sync(_))
    unsafeRun(for {
      sequence1 <- ios.collect
      sequence2 <- IO.collect(ios)
    } yield sequence1 must ===(sequence2))
  }
```
I get this error
```markdown
[error] [E1] core/jvm/src/test/scala/scalaz/zio/IOSyntaxSpec.scala
[error]      missing argument list for method collect in class List
[error]      Unapplied methods are only converted to functions when a function type is expected.
[error]      You can make this conversion explicit by writing `collect _` or `collect(_)(_)` instead of `collect`.
[error]      L209:       sequence1 <- ios.collect
[error]                                   ^
```

it seems there are naming clashes with other extension methods:

![image](https://user-images.githubusercontent.com/9019485/50251185-5e88e480-03e3-11e9-8d37-7a25f82c0e43.png)


btw this works as intended
```scala
def t4 = {
    val ios = TestData.map(IO.sync(_))
    unsafeRun(for {
      sequence1 <- ioIterableSyntax(ios).collect
      sequence2 <- IO.collect(ios)
    } yield sequence1 must ===(sequence2))
  }
```

Reply:
It looks like it's finding the can build from one.

Let's fix this by using `collectAll` and `collectAllPar`, rather than `collectSeq` and `collectPar`. In all other cases, we let `seq` be the default, and `par` require the suffix. So I'd like to do the same here for consistency.

Clearly `collect` is too common.

Reply:
:heavy_check_mark: